test/linalg/bunchkaufman.jl

```diff
@@ -11,7 +11,7 @@ n = 10
 n1 = div(n, 2)
 n2 = 2*n1
 
-srand(1234321)
+srand(1234322)
```

Comment:
I kept getting this error https://travis-ci.org/JuliaLang/julia/jobs/132724032#L1787-L1808 which I cannot reproduce locally and which should be completely unrelated to the PR so I wanted to see if anything changed with a new seed.